### PR TITLE
Disable white screen when loading atone form + Update README in Japanese version

### PR DESCRIPTION
**Pull request by at-hoaiphan**
#### Github Issue URL/Tags `(required)`
- #47

#### What's this PR do? Please give a technical summary `(required)`
- Make listener getting event when atone form loading success, then disable white screen

#### Any references (stackoverflow …) `(optional)`

### CODE REVIEW

**Comment by at-hoaiphan** [comment]
@at-kietvo I checked on the devices in android version 4.x, it's OK. Please check my PR. Thank you.

File: atonelibrary/src/main/java/atone/asiantech/vn/atonelibrary/OnFormRenderListener.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package atone.asiantech.vn.atonelibrary;
+
+/**
+ * An interface gets callback from JavaScriptInterface when form loading success in web-view.
+ */
+
```
Comment (at-kietvo) [inline atonelibrary/src/main/java/atone/asiantech/vn/atonelibrary/OnFormRenderListener.java]: Please remove this line.

**Comment by at-hoaiphan** [comment]
@at-kietvo I fixed it, please check again. Thank you

**Comment by at-hoaiphan** [comment]
@at-kietvo I fixed and update version. Please check my PR again. Thank you.

File: README.en.md [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+==================
+## A. Requirement
+------
+- Android 4.0+
+- Android Studio
+## B. Installation
+------
+- Add `dependency` to `build.gradle`.
+```
+compile 'jp.co.netprotections:atonelibrary:1.0.0'
```
Comment (at-kietvo) [inline README.en.md]: Do you think we should upgrade to 1.0.1 ? and English also

**Comment by at-hoaiphan** [comment]
@at-kietvo As client's requirement, they want us upload README in both English and Japanes.
In this version, we fixed bug of version 1.0.0 that is has a white screen when load atone form. So I think we should upgrade to 1.0.1

**Pull request merged**